common-rx/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+
+dependencies {
+    // noinspection DifferentStdlibGradleVersion
+    implementation(
+        "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    )
+
+    api(
+        "io.reactivex.rxjava2:rxjava:$rxjava_version",
+        "io.reactivex.rxjava2:rxkotlin:$rxkotlin_version"
+    )
+}
```

Comment:
Hmmm, does this we can remove the  libraries from `data/build.gradle`? :still_thinking:

Most likely `"io.reactivex.rxjava2:rxandroid:$rxandroid_version"` should be moved in to `common-rx/build.gradle` as well 🙏 

Sorry of this is out of scope 🙈

Reply:
@Tuubz correct, @manh-t we could update in next PR https://github.com/nimblehq/android-templates/pull/48

Reply:
@luongvo 🆗